From e8112cd4bafdbbfd88f3cc9a10b7b8e08fbbea01 Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Tue, 24 Jan 2012 12:40:29 +0100
Subject: [PATCH 4/4] [ARM] Don't forcefully disable avahi, configure has checks for it already

This fixes zeroconf on ARM platforms that do support avahi like Beagleboard/Angstrom

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---

Upstream-status: Pending

 xbmc/system.h |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/xbmc/system.h b/xbmc/system.h
index c838dc8..b10ebae 100644
--- a/xbmc/system.h
+++ b/xbmc/system.h
@@ -222,8 +222,6 @@
 
 // ARM does not support certain features... disable them here!
 #ifdef _ARMEL
-#undef HAS_AVAHI
-#undef HAS_ZEROCONF
 #undef HAS_VISUALISATION
 #undef HAS_FILESYSTEM_HTSP
 #endif
-- 
1.7.2.5

